web/src/main/java/nl/tehdreamteam/se42/web/command/Command.java

```diff
@@ -0,0 +1,38 @@
+package nl.tehdreamteam.se42.web.command;
+
+/**
+ * Represents a single {@code Command} that can be executed.
+ */
+public abstract class Command {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "Command" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1610)